Source/DafnyCore/Backends/CSharp/CsharpCodeGenerator.cs

```diff
@@ -2037,7 +2035,7 @@ protected class ClassWriter : IClassWriter {
     protected override void EmitHalt(IToken tok, Expression/*?*/ messageExpr, ConcreteSyntaxTree wr) {
       var exceptionMessage = Expr(messageExpr, false, wr.Fork());
       if (tok != null) {
-        exceptionMessage.Prepend(new LineSegment(SymbolDisplay.FormatLiteral(tok.TokenToString(Options) + ": ", true) + " + "));
+        exceptionMessage.Prepend(new LineSegment($"\"{tok.TokenToString(Options)}: \" + "));
```

Comment:
Why? I would think FormatLiteral would be better because it would do escaping?

Reply:
I removed a Roslyn dll dependency, so `SymbolDisplay` was no longer available.